Resources/Prototypes/Entities/Objects/Misc/business card.yml

```diff
@@ -0,0 +1,33 @@
+- type: entity
+  id: SyndicateBusinessCard
+  name: Syndicate business card
+  parent: BaseItem
```

Comment:
This should just inherit from the Paper entity in paper.yml and only overwrite what needs to be changed.